src/views/about/index.js

```diff
@@ -12,9 +29,132 @@ import {Typography} from '@material-ui/core';
  */
 const About = () => {
   return (
-    <div data-testid="about">
-      <Typography variant="h1">Enamel Dashboard</Typography>
-    </div>
+    <Container data-testid="about">
+      <TitleContainer>
+        <Logo src="/images/chromium.png" alt="Chrome Security UX" />
+        <TitleTextContainer>
+          <Typography variant="h2">Chrome Security UX</Typography>
+          <Typography variant="h6">Supporting Chrome Deprecations</Typography>
+        </TitleTextContainer>
+      </TitleContainer>
+      <Accordion defaultExpanded>
+        <AccordionSummary expandIcon={<ExpandMoreIcon />}>
+          <Typography variant="subtitle2">
+            Why is the Chrome Security Dashboard needed?
+          </Typography>
+        </AccordionSummary>
+        <AccordionDetails>
+          <AccordionTextContainer>
+            <Typography gutterBottom variant="body1">
+              The Chrome Security UX team often needs changes that, while
+              security positive, can cause breakage of web content (e.g. mixed
+              content, HTTP vs HTTPs, forms, etc).
+            </Typography>
+            <Typography gutterBottom variant="body1">
+              Because of that, it is a problem when Chrome is going to launch a
+              new feature or deprecate any functionality. That&lsquo;s why we
+              aim to provide useful data to measure breakage before rolling out
+              changes, to see if benefit is worth it.
+            </Typography>
+            <Typography gutterBottom variant="body1">
+              The Chrome Security Dashboard is intended to present public data
+              about changes to be done in future Chrome Versions, and to serve
+              as a tool to negotiate next steps with other browsers.
+            </Typography>
+          </AccordionTextContainer>
+          <Icon alt="Protection" src="images/protection.png" />
+        </AccordionDetails>
+      </Accordion>
+      <Accordion defaultExpanded>
+        <AccordionSummary expandIcon={<ExpandMoreIcon />}>
+          <Typography variant="subtitle2">
+            Where do we get the data from?
+          </Typography>
+        </AccordionSummary>
+        <AccordionDetails>
+          <AccordionTextContainer>
+            <Typography gutterBottom variant="body1">
+              As part of our effort to show enough data to support decisions, we
+              evaluated many options of public data sources and ended up using
+              HTTP Archive public datasets hosted on BigQuery service on Google
```

Comment:
I would remove the "hosted on BigQuery service on Google Cloud" part, since for a general user, the meaningful part is what dataset it's from, not so much how it's retrieved.

Reply:
Done!